Content.IntegrationTests/Tests/Fluids/PuddleTest.cs

```diff
@@ -67,7 +67,8 @@ public async Task SpaceNoPuddleTest()
 
             await server.WaitAssertion(() =>
             {
-                var coordinates = grid.ToCoordinates();
+                var gridOwner = grid.Owner;
+                var coordinates = gridOwner.ToCoordinates();
```

Comment:
Since this is only used here, you can just make it one variable.
```suggestion
                var coordinates = grid.Owner.ToCoordinates();
```